src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Debug/IVersionedLaunchSettingsProvider.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using Microsoft.VisualStudio.Composition;
+
+namespace Microsoft.VisualStudio.ProjectSystem.Debug
+{
+    /// <summary>
+    /// A variant of <see cref="ILaunchSettingsProvider.SourceBlock"/> that provides
```

Comment:
Did you mean:

```suggestion
    /// A variant of <see cref="ILaunchSettingsProvider"/> that provides
```

Reply:
No, because right now it's really only a replacement for the one method, not the entire interface. If I update the new interface to derive from `ILaunchSettingsProvider` then I will update this accordingly.

Reply:
Updated as part of changing the derivation.